src/data/footer.yaml

```diff
@@ -124,6 +124,8 @@ footer:
       #   blank: false
       - title: Privacy policy
         url: "/privacy-policy/"
+      - title: Gestione cookie
+        url: "/privacy-policy/#gestione-cookie"
```

Comment:
This doesn't land you on the right section because there's no id for it, but I guess it's tracked by #919.

Reply:
This is working well right now, there was a typo days ago on privacy-policy yaml.